server/drivers/postgres/index.js

```diff
@@ -29,6 +30,7 @@ class Client {
       port: connection.port || undefined,
       ssl: connection.postgresSsl,
       stream: createSocksConnection(connection),
+      query_timeout: getTimeoutSeconds() * 1000,
```

Comment:
Instead of sourcing this from app config, can this instead be added to postgres fields? This would allow setting different timeouts for different connections, and have it exposed via UI.

To do this, you can add the following to the `fields` array at bottom of the file:
```
{
    key: 'queryTimeout',
    formType: 'TEXT',
    label: 'Query Timeout (seconds)',
  }
  ```
  
and then access that value when creating the connection with `connection.queryTimeout`

Reply:
Yes I'm just wondering how to connect it to the env variable SQLPAD_TIMEOUT_SECONDS. Which is the simplest to change.

Reply:
We want to enforce this timeout so that even "malicious users" (or junior developers of our infrastructure) cannot make too long running queries

Reply:
Ok I think I get it